src/popup/router/pages/Networks.vue

```diff
@@ -59,6 +59,18 @@
         v-model="network.compilerUrl"
         data-cy="compiler"
       />
+      <div class="text-left expand" @click="expanded = !expanded">
+        <img :class="expanded ? 'open' : ''" src="../../../icons/carret-down.svg" />
```

Comment:
```suggestion
        <img :class="{ 'open' : expanded }" src="../../../icons/carret-down.svg" />
```
or even to rename class to `expanded`, because they mean the same here